### PR TITLE
UIEH-962 Choose custom package - batch load packages

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Load packages for package select field when user searches by package name.

## Screenshots
image

## Related PRs
https://github.com/folio-org/stripes-components/pull/1586

## Issues
[UIEH-962](https://issues.folio.org/browse/UIEH-962)

### CODE REVIEW

**Review by mariiaschv — APPROVED** [review]

File: src/components/title/_fields/package-select/package-select-field.js [hunk 1]
```diff
@@ -42,6 +62,8 @@ const PackageSelectField = ({ options }) => {
 };
 
 PackageSelectField.propTypes = {
+  loadingOptions: PropTypes.bool,
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/_fields/package-select/package-select-field.js:65]: Please, add default prop for `loadingOptions`

File: src/components/title/_fields/package-select/package-select-field.test.js [hunk 2]
```diff
@@ -1,41 +1,70 @@
 import { Form } from 'react-final-form';
-import { render } from '@testing-library/react';
+import { fireEvent, render } from '@testing-library/react';
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/_fields/package-select/package-select-field.test.js:2]:
```suggestion
import {
  fireEvent,
  render,
} from '@testing-library/react';
```

File: src/components/title/_fields/package-select/package-select-field.test.js [hunk 3]
```diff
@@ -1,41 +1,70 @@
...
   it('should render package select field', () => {
-    const { getByText } = renderPackageSelectField([{
-      label: 'label1',
-      key: 'key1',
-    }]);
+    const { getByText } = renderPackageSelectField({
+      options: [{
+        label: 'label1',
+        key: 'key1',
+      }]
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/_fields/package-select/package-select-field.test.js:27]:
```suggestion
      }],
```

File: src/components/title/_fields/package-select/package-select-field.test.js [hunk 4]
```diff
@@ -1,41 +1,70 @@
...
+  it('should debounce onFilter function', async () => {
+    const { getByLabelText } = renderPackageSelectField({
+      options: [{
+        label: 'label1',
+        key: 'key1',
+      }],
+    });
+
+    const input = getByLabelText('stripes-components.selection.filterOptionsLabel');
+    fireEvent.change(input, { target: { value: 'packageName' } });
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/_fields/package-select/package-select-field.test.js:65]:
```suggestion

    fireEvent.change(input, { target: { value: 'packageName' } });
```
empty line after `const`

**Comment by Vladyslav-Velytskyi** [comment]
Do we have stories to cover routes by tests?

**Comment by BogdanDenis** [comment]
> Do we have stories to cover routes by tests?

No, we only have stories for testing view components

**Pull request merged**